requirements.txt

```diff
@@ -10,7 +10,7 @@ toml~=0.10
 appdirs~=1.4
 semantic_version~=2.10
 dask[delayed]~=2022.4.1
-autoray==0.3.1
+autoray>=0.3.1,<=0.6.3
```

Comment:
I wonder why we are doing `~=` pinning here, I thought the preference was `==` pinning for `requirements.txt` to define a more fixed development environment.

Reply:
Not sure 🤔